src/containers/StructurePage/folderComponents/SettingsMenuDropdownType.tsx

```diff
@@ -81,7 +81,13 @@ const SettingsMenuDropdownType = ({ rootNodeId, node, onCurrentNodeChanged, node
               })}
             </TabsTrigger>
             <TabsTrigger value="toggleMetadataVisibility">{t("metadata.changeVisibility")}</TabsTrigger>
-            {isTaxonomyAdmin && <TabsTrigger value="deleteTopic">{t("taxonomy.deleteNode")}</TabsTrigger>}
+            {isTaxonomyAdmin && (
+              <TabsTrigger value="deleteSubject">
```

Comment:
No har du samme tabsTrigger to plasser.

Reply:
Har ikke noe å si, men kan jo oppdatere til programme liksågreit!